recipes/pygeoif/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "0.6" %}
+
+package:
+    name: pygeoif
+    version: {{ version }}
+
+source:
+    fn: pygeoif-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/source/p/pygeoif/pygeoif-{{ version }}.tar.gz
+    md5: e5d423724f9894aa2b2a1a8eb2314051
+
+build:
+    number: 0
+    preserve_egg_dir: True
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+    run:
+        - python
+
+test:
+    imports:
+        - pygeoif
+    requires:
+        - pytest
+
+about:
+    home: https://github.com/cleder/pygeoif/
+    license: GNU Library or Lesser General Public License (LGPL)
```

Comment:
This one I could not find. That is all I got from them:

https://github.com/cleder/pygeoif/blob/master/setup.py#L51